.env.example

```
@@ -62,11 +62,20 @@ CHATBOT_INTERNAL_URL=http://chatbot:5007
 FLASK_SECRET_KEY=
 FLASK_PORT=5080
 AUTH_SECRET=
+
+# Secrets backend: "vault" (default) or "aws_secrets_manager"
+# SECRETS_BACKEND=vault
+
+# HashiCorp Vault (default backend)
 VAULT_TOKEN=
 VAULT_ADDR=http://vault:8200
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VAULT_ADDR key should go before the VAULT_TOKEN key

(UnorderedKey)